specification/resourcemover/resource-manager/readme.java.md

```diff
@@ -0,0 +1,33 @@
+## Java
+
+These settings apply only when `--java` is specified on the command line.
+Please also specify `--azure-libraries-for-java-folder=<path to the root directory of your azure-libraries-for-java clone>`.
+
+``` yaml $(java)
+azure-arm: true
+fluent: true
+namespace: com.microsoft.azure.management.migrate.resourcemover
```

Comment:
Currently Java SDK will use only one word in namespace. like `**management.resourcemover` rather the the above.
Please remove all `migrate` in java.

Reply:
Removed all references to "migrate" in java - readme

Reply:
Then Java should be fine